carma-messenger-core/carma-messenger/launch/plugins.launch

```diff
@@ -17,6 +17,8 @@
 <!--
 	Launch file for launching the use case specific nodes being used in carma-messenger
 -->
-<launch>
-  
+<launch>  
+  <!-- Truck Inspection Plugin -->
+    <rosparam command="load" file="$(find truck_inspection_plugin)/config/parameters.yaml"/>
```

Comment:
This line should not be needed because it is already done here
https://github.com/usdot-fhwa-stol/carma-messenger/blob/bugfix/add_truck_inspection_plugin_to_messenger_core/carma-messenger-core/truck_inspection_plugin/launch/truck_inspection_plugin.launch

Reply:
It is updated. Please see this commit:
https://github.com/usdot-fhwa-stol/carma-messenger/commit/f7e6dcf3b518961f1493992815f5dc3c852908c6